src/main/java/org/fundacion/jala/converter/models/facade/ZipFileFacade.java

```diff
@@ -55,18 +55,19 @@ public static void getZipFileVideo(final ParameterOutputChecksum parameterOutput
     }
 
     /**
-     * Compresses files.
+     * Compresses any files.
      *
      * @param parameterOutputChecksum object of ParameterOutputChecksum.
-     * @param metadata if add metadata into zip.
-     * @param storagePath path and name of file to compress.
+     * @param metadata is a parameter if add metadata of video into zip.
+     * @param storagePath is the path's name of file to compress.
      * @param thumbnail if add thumbnail of video into zip.
      * @throws IOException when invalid path is given in zipFiles.
      * @throws InterruptedException is exception if process is interrupted.
      */
-    private static void getZipFile(final ParameterOutputChecksum parameterOutputChecksum, final String metadata,
-                                   final String storagePath, final boolean thumbnail)
-            throws IOException, InterruptedException {
+    private static void getZipFile(final ParameterOutputChecksum parameterOutputChecksum,
+                                   final String metadata,
```

Comment:
There should not be one parameter per line, only if it passes 120 characters

Reply:
ok. It was solved